screens/HoursScreen.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import {
+  ScrollView,
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+import * as firebase from 'firebase';
+import Days from '../constants/Days';
+import Icons from '../constants/Icons';
+
+export default class HoursScreen extends React.Component {
+  static navigationOptions = {
+    header: null
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      daysToHours: {},
+    };
+  }
+
+  async componentDidMount() {
+    var daysToHours = {};
+    await firebase
+      .database()
+      .ref('info')
+      .once('value')
+      .then(function(data) {
+        const DB_DAYS_KEYS_TO_READABLE_DAYS = Days.DB_DAYS_KEYS_TO_READABLE_DAYS;
+        data.forEach(function(childNode) {
+          var dayKey = childNode.key;
+          if (DB_DAYS_KEYS_TO_READABLE_DAYS.hasOwnProperty(dayKey)) {
+            var day = childNode.val();
+            var readableDay = DB_DAYS_KEYS_TO_READABLE_DAYS[dayKey];
+            daysToHours[readableDay] = {
+              day: readableDay,
+              hours: day['12hours'],
+              restock: day['restock'],
+            };
+          }
+        });
+        this.setState({
+          daysToHours: daysToHours
+        });
+      }.bind(this));
+  }
+
+  getEmojis(restock) {
+    const emojis = Icons.EMOJIS;
+    return Object.keys(emojis).map(function(key) {
+      if (restock[key] == 1) {
+        return emojis[key];
+      }
+    });
+  }
+
+  renderHours() {
+    var orderedDaysToHours = Days.ORDERED_READABLE_DAYS.map((readableDay) => {
+      return this.state.daysToHours[readableDay];
+    });
+    return orderedDaysToHours.map((dayObject) => {
+      if (dayObject) {
+        return (
```

Comment:
Are the parentheses necessarily? Not sure -- can you try without em?

Reply:
necessary